### PR TITLE
Example Object endpoint

### PR DESCRIPTION
**Pull request by kamaldodiya**
Description

Implement new endpoint example_objects which will return list of objects containing  gene, location, variant example

Related JIRA

https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2221

See it in Action

```
curl 'https://staging-2020.ensembl.org/api/metadata/genome/a7335667-93e7-11ec-a39d-005056b38ce3/example_objects' | jq
```

Please note  data on _stage is out of date as compared to stage-1 so once it is synced it the actual data will appear
Local using stage-1

```
curl 'http://localhost:8014/api/metadata/genome/a7335667-93e7-11ec-a39d-005056b38ce3/example_objects' \| jq
[
    {
        "type": "gene",
        "id": "ENSG00000221914"
    },
    {
        "type": "location",
        "id": "8:26291508-26372680"
    }
]
```

### CODE REVIEW

**Review by azangru — APPROVED** [review]

**Comment by azangru** [comment]
The words "statistics" and "stats" are used a lot in this PR. This will probably be confusing to future readers. Example objects aren't statistics. The fact that they are stored in the database alongside species statistics is a coincidence.

Doesn't production call them "attributes"? Would `for stats_item in samples_input` read better as `for attribute in genome_attributes`?

File: app/api/models/statistics.py [hunk 1]
```diff
@@ -140,3 +141,36 @@ def __init__(self, **data):
...
+        values["example_objects"] = extracted_example_objects
+        return values
+
+    def extract_samples(samples_input):
+        extracted_samples = []
+        try:
+            for stats_item in samples_input:
+                try:
+                    if stats_item["name"] == "genebuild.sample_gene":
+                        e_g = ExampleObject(type="gene", id=stats_item["statisticValue"])
```
Comment (Mehrnaz-Charkhchi) [inline app/api/models/statistics.py:164]: It would more clear to use full names and not abbreviations.

File: app/api/models/statistics.py [hunk 2]
```diff
@@ -140,3 +141,36 @@ def __init__(self, **data):
         data["regulation_stats"] = data["_compiled_data"]
 
         super().__init__(**data)
+
+class ExampleObject(BaseModel):
+    type: str
+    id: str
+
+class ExampleObjects(BaseModel):
```
Comment (Mehrnaz-Charkhchi) [inline app/api/models/statistics.py:149]: It's recommended to use singular format for Model class names and in cases like this using something ExampleObjectList . 
At the first glance I didn't even notice plural s

**Comment by kamaldodiya** [comment]
> Doesn't production call them "attributes"? Would for stats_item in samples_input read better as for attribute in genome_attributes?

Renamed

> The fact that they are stored in the database alongside species statistics is a coincidence (they could also be fetched through the https://github.com/Ensembl/ensembl-metadata-service/pull/33, where they wouldn't be called statistics).

At the moment all I get is this and there is no data which we need 

```
genome_uuid: "a7335667-93e7-11ec-a39d-005056b38ce3"
assembly {
  accession: "GCA_000001405.29"
  name: "GRCh38.p14"
  ucsc_name: "hg38"
  level: "chromosome"
  ensembl_name: "GRCh38.p14"
  assembly_uuid: "fd7fea38-981a-4d73-a879-6f9daef86f08"
  is_reference: true
  url_name: "grch38"
}
taxon {
  taxonomy_id: 9606
  scientific_name: "Homo sapiens"
}
created: "2023-09-22 15:04:45"
organism {
  common_name: "human"
  scientific_name: "Homo sapiens"
  ensembl_name: "homo_sapiens"
  scientific_parlance_name: "Human"
  organism_uuid: "1d336185-affe-4a91-85bb-04ebd73cbb56"
  taxonomy_id: 9606
  species_taxonomy_id: 9606
}
attributes_info {
  assembly_level: "chromosome"
  assembly_date: "2013-12"
  assembly_provider_name: "Genome Reference Consortium"
  assembly_provider_url: "https://www.ncbi.nlm.nih.gov/grc"
}
related_assemblies_count: 99
release {
  release_date: "Unreleased"
  release_label: "Unreleased"
  site_name: "Unknown (not released yet)"
  site_label: "Unknown (not released yet)"
  site_uri: "Unknown (not released yet)"
}

```

The only place I am getting is stats procedure and that is why it is being used for getting example_objects.

**Review by azangru — APPROVED** [review]

**Review by Mehrnaz-Charkhchi — APPROVED** [review]

**Pull request merged**